x-pack/filebeat/input/cometd/cometd_integration_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
```

Comment:
```suggestion
//go:build integration
// +build integration
```

Add integration test build tags to exclude this from running with the unit tests. To run the integration tests use `go test -tags integration`

Reply:
Right